src/transformStateWithClones.js

```diff
@@ -7,7 +7,55 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformState = [];
+  let stateCopy = { ...state };
+
+  for (const act of actions) {
```

Comment:
Add more meaningful variable name
```suggestion
  for (const action of actions) {
```